source/dbo/Stored Procedures/Power BI Reporting/uspUpdateRecordPostcode.sql

```diff
@@ -0,0 +1,42 @@
+﻿/*
+	This procedure creates a cleaned postcode from the value supplied from the notification system (ETS or NTBS)
+	Provided the postcode matches the format and length of a UK postcode, this will insert a single space 3 characters from
+	the left hand side of the string
```

Comment:
I think the stored procedure is correct, but the comment should say "from the right hand side of the string"